code/modules/clothing/spacesuits/rig/suits/station.dm

```diff
@@ -162,6 +166,61 @@ Advanced Voidsuit: Technomancer Exultant
 	desc = "Advanced magnetic boots that have a lighter magnetic pull, placing less burden on the wearer."
 	mag_slow = 1
 
+/***************************************
+Technomancer RIG
+***************************************/
+/obj/item/weapon/rig/techno
+	name = "technomancer RIG suit control module"
+	suit_type = "technomancer RIG suit"
+	desc = "An advanced RIG suit that protects against hazardous, low pressure and high temperature environments."
+	icon_state = "techno_rig"
+	rarity_value = 20
+	armor = list(
+		melee = 30,
+		bullet = 30,
+		energy = 30,
+		bomb = 50,
+		bio = 100,
+		rad = 100
+	)
+	slowdown = 1
+	drain = 3
+	offline_slowdown = 3
+	offline_vision_restriction = 0
+
+	max_heat_protection_temperature = FIRESUIT_MAX_HEAT_PROTECTION_TEMPERATURE
+
+	helm_type = /obj/item/clothing/head/space/rig/techno
+	glove_type = /obj/item/clothing/gloves/rig/techno
+	boot_type = /obj/item/clothing/shoes/magboots/rig/techno
+
+	extra_allowed = list(
+		/obj/item/weapon/storage/toolbox,
+		/obj/item/weapon/storage/briefcase/inflatable,
+		/obj/item/device/t_scanner,
+		/obj/item/weapon/rcd
+	)
+
+	spawn_blacklisted = TRUE
+
+/obj/item/weapon/rig/techno/equipped
+	initial_modules = list(
+		/obj/item/rig_module/storage,
+		/obj/item/rig_module/maneuvering_jets,
+		)
+
+/obj/item/clothing/gloves/rig/techno
+	name = "insulated gloves"
+	siemens_coefficient = 0
+
+
+/obj/item/clothing/shoes/magboots/rig/techno
+	name = "Advanced Magboots"
```

Comment:
```suggestion
	name = "advanced magboots"
```
We should avoid uppercases if they're not necessary.
~~shut up i know it's also on TM RIG~~